endpoints/HandoverEndpoint.ts

```diff
@@ -0,0 +1,40 @@
+import { HttpStatusCode, IHttp, IModify, IPersistence, IRead } from '@rocket.chat/apps-engine/definition/accessors';
+import { ApiEndpoint, IApiEndpointInfo, IApiRequest, IApiResponse } from '@rocket.chat/apps-engine/definition/api';
+import { ILivechatRoom } from '@rocket.chat/apps-engine/definition/livechat';
+import { Logs } from '../enum/Logs';
+import { performHandover } from '../helperFunctions/HandoverHelpers';
+import { createHttpResponse } from '../helperFunctions/HttpHelpers';
+
+export class HandoverEndpoint extends ApiEndpoint {
```

Comment:
Can you add documentation for using these endpoints, and maybe list them in the app setting page like Murtaza has [here?](https://github.com/RocketChat/Apps.Dialogflow#dialogflows-api)

Reply:
Yeah, I am working on updating readme right now, that includes adding endpoint documentation along with some other new changes. ✌️

Reply:
They are also already listed on App Setting page. If it is not showing up, just uninstall and install the app again. 👍

Reply:
Updated the readme 😄